docs/resources/event_gateway_backend_cluster.md

```diff
@@ -15,32 +15,32 @@ EventGatewayBackendCluster Resource
 ```terraform
 resource "konnect_event_gateway_backend_cluster" "my_eventgatewaybackendcluster" {
   provider = konnect-beta
-authentication = {
-sasl_scram = {
-algorithm = "sha256"
-password = "${vault.env['MY_ENV_VAR']}"
-username = "...my_username..."
-}
-}
-bootstrap_servers = [
+  authentication = {
+    sasl_scram = {
+      algorithm = "sha256"
+      password  = "$${vault.env['MY_ENV_VAR']}"
```

Comment:
Use of extra "$" - it is an escape sequence for terraform - https://developer.hashicorp.com/terraform/language/expressions/strings

We have used it tests too - https://github.com/Kong/terraform-provider-konnect-beta/blob/main/tests/resources/testdata/TestEventGateway/smoke-test/main.tf#L32